docs/journeys/journey-launcher.md

```diff
@@ -0,0 +1,235 @@
+---
+sidebar_position: 5
+---
+
+# Journey Launcher
+
+The Journey Launcher allows journeys to share a common start screen. When embedded in a client web page, the journey launcher allows the user to select from various journeys shown as tabs. The first step of each journey is shown whenever the corresponding journey tab is selected.
+
+In order to setup a journey launcher, follow the steps below:
+
+# 1. Get the ID for the Journey Launcher template
+
+You need to have access to a Journey launcher tamplate in order to create a Launcher Journey. To get the templateId of the launcher template, use the following request:
```

Comment:
**Suggestion:**

You need access to a Journey Launcher template to create a Launcher Journey. To get the `templateId` of the launcher template, use the following request: